lib/components/src/typography/DocumentFormatting.tsx

```diff
@@ -302,6 +302,12 @@ export const LI = styled.li<{}>(withReset, ({ theme }) => ({
     marginTop: '.25em',
     marginBottom: 0,
   },
+  '& ul': {
+    listStyle: 'disc',
+  },
+  '& ol': {
+    listStyle: 'decimal',
+  },
```

Comment:
This causes:
https://www.chromatic.com/snapshot?appId=5a375b97f4b14f0020b0cda3&id=5ef26a5d350d7d0022a27d4f

But I'm fine with it actually